korman/exporter/animation.py

```diff
@@ -89,22 +93,27 @@ def get_ranges(*args, **kwargs):
                 if i is not None:
                     yield i.frame_range[index]
         atcanim.name = "(Entire Animation)"
-        atcanim.start = self._convert_frame_time(min(get_ranges(obj_action, data_action, index=0)))
-        atcanim.end = self._convert_frame_time(max(get_ranges(obj_action, data_action, index=1)))
+        def anim_range_type(plAGAnim):
```

Comment:
You've turned this into a function, and it's not being called anywhere, so I don't think this will export correctly. I think the start, end, and "FIXME" stuff should all go back to the way it was before this change request. Then you can add in right below `atcanim.end = ...`:

```python
if isinstance(atcanim, plAgeGlobalAnim):
    atcanim.globalVarName = anim_mod.obj_sdl_anim
```